homeassistant/components/climate/ecobee.py

```diff
@@ -204,6 +202,18 @@ def is_away_mode_on(self):
                 break
         return 'away' in mode
 
+    @property
```

Comment:
If this is wrong for the platform, it means all platforms have this fault, and it should be fixed in `__init__.py`